Tries to install very old version of libxml2.  Go with the system
libxml2 despite the very serious sounding warnings.

--- alienfile.orig	2025-03-19 13:43:22 UTC
+++ alienfile
@@ -65,7 +65,6 @@ my %bad_versions = map { $_ => 1 } (
   #  The next set have reported CVEs
   (map {'2.11.' . $_} (0 .. 9)),
   (map {'2.12.' . $_} (0 .. 9)),
-  (map {'2.13.' . $_} (0 .. 6)),
 );
 
 meta->around_hook(
